answerking_app/urls/item_urls.py

```diff
@@ -7,7 +7,7 @@
 urlpatterns: list[partial] = [
     path("items", item_views.ItemListView.as_view(), name="item_list"),
     path(
-        "items/<item_id>",
+        "items/<int:pk>",
```

Comment:
when you do it like this does it return the pk as an int or as a string?

Reply:
as int